spec/model/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe User, type: :model do
```

Comment:
- [ ] It is a requirement that Model specs has **Validation** tests. Kindly implement this.
-For example:

```
describe 'validations' do
  it { should validate_presence_of(:name) }
end

```

- [ ] Kindly ensure to implement tests for all the validations in the `user` model,